flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -170,14 +172,15 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
   Future<void> initialize() async {
     if (_isDisposed) {
       throw Exception(
-          'initialize was called on a disposed VlcPlayerController');
+        'initialize was called on a disposed VlcPlayerController',
+      );
     }
     if (value.isInitialized) {
       throw Exception('Already Initialized');
     }
 
     _lifeCycleObserver = VlcAppLifeCycleObserver(this);
-    _lifeCycleObserver!.initialize();
+    _lifeCycleObserver?.initialize();
```

Comment:
Maybe this will look better?
```suggestion
    ..initialize();
```